metadata-service/plugin/src/main/java/com/datahub/plugins/common/IsolatedClassLoader.java

```diff
@@ -0,0 +1,10 @@
+package com.datahub.plugins.common;
+
+import com.datahub.plugins.Plugin;
+import javax.annotation.Nonnull;
+
+
+public interface IsolatedClassLoader {
```

Comment:
Let's add java docs to this interface. What it is for? Why is it necessary?

Reply:
Also the name "IsolatedClassLoader" does not really align with the name of the method "instantiatePlugin"